demo/api/src/db/ormconfig.cli.ts

```diff
@@ -1,12 +1,17 @@
+import { createFileEntity, createFolderEntity, FileImage } from "@comet/cms-api";
 import { Options } from "@mikro-orm/core";
+import { DamScope } from "@src/dam/dto/dam-scope";
 import { PageTreeNodeScope } from "@src/page-tree/dto/page-tree-node-scope";
 
 import ormConfig from "./ormconfig";
 
+const Folder = createFolderEntity({ Scope: DamScope });
+const File = createFileEntity({ Scope: DamScope, Folder });
+
 const config: Options = {
     ...ormConfig,
-    entities: ["./dist/**/*.entity.js", PageTreeNodeScope],
-    entitiesTs: ["./src/**/*.entity.ts", PageTreeNodeScope],
+    entities: ["./dist/**/*.entity.js", PageTreeNodeScope, File, FileImage, Folder],
```

Comment:
I've added them to generate the migrations, might remove them afterwards. I'm not sure if the change to the entity (added scope) is picked up by the CLI when the entities are not present, however. This is a problem for all library entities I believe

Reply:
if it doesn't have to be done like that in the application, then I don't see a problem.

Reply:
I've just tested it. Without the entities specified, changes the scope object are not picked up by the CLI. Even worse, the generated migration contains Drop Statements for the DAM tables (this might be adverted by setting `migrations.dropTables` to `false` in the ORM config.

BTW, it doesn't work for the Redirect entity either.

Reply:
in application/demo? This sucks. We have to do that differently, also for redirects.

idea, if we create the entity and use the factory there, mikro-orm will find it:
app/src/redirects/redirect.entity.ts:
`export const Redirect = RedirectEntityFactory.create({ customTargets, Scope });`

Reply:
> idea, if we create the entity and use the factory there, mikro-orm will find it:
app/src/redirects/redirect.entity.ts:

I like this idea!

Reply:
Done in 4bc9423765c8816a72ab00fb394179fd5c3e5897